rebasehelper/exceptions.py

```diff
@@ -26,7 +26,21 @@ class RebaseHelperError(Exception):
     Class representing Error raised inside rebase-helper after intentionally
     catching some expected and well known exception/error.
     """
-    pass
+
+    def __init__(self, *args, **kwargs):
+        """
+        Init method for RebaseHelperError exception
+
+        :param args: error message
+        :param kwargs: more optional arguments
+        """
+        if not args:
+            self.msg = None
+        elif len(args) > 1:
+            self.msg = args[0] % args[1:]
+        else:
+            self.msg = args[0]
+        self.logfile = kwargs.get('logfile')
```

Comment:
Also, since there is now `RebaseHelperError.msg` attribute, you could use it in [`CliHelper`](https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/cli.py#L259-L263).